lib/src/components/NxTableCell/NxTableCell.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import classnames from 'classnames';
+import { faSort, faSortDown, faSortUp } from '@fortawesome/free-solid-svg-icons';
+
+import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
+
+import { Props } from './types';
+export {Props} from './types';
```

Comment:
Minor: unbalanced parenthesis spacing. 

```suggestion
export { Props } from './types';
```